website/content/docs/beta-features.md

```diff
@@ -647,3 +650,19 @@ content
         │   └── index.md
         └── index.md
 ```
+
+## Remark plugins
+
+You can register plugins to customize [`remark`](https://github.com/remarkjs/remark), the library used by the richtext editor for serializing and deserializing markdown.
+
+```js
+import { registerRemarkPlugin } from 'netlify-cms-core';
+
+// register a plugin
+registerRemarkPlugin(plugin);
+
+// provide global settings to all plugins, e.g. for customizing `remark-stringify`
+registerRemarkPlugin({ settings: { bullet: '-' } });
```

Comment:
```suggestion
// register a plugin
CMS.registerRemarkPlugin(plugin);

// provide global settings to all plugins, e.g. for customizing `remark-stringify`
CMS.registerRemarkPlugin({ settings: { bullet: '-' } });
```

To match other parts of the docs:
https://www.netlifycms.org/docs/custom-widgets/#registerwidget
https://www.netlifycms.org/docs/beta-features/#registering-to-cms-events